PlayCover/Views/App Views/PlayAppView.swift

```diff
@@ -182,28 +182,30 @@ struct PlayAppConditionalView: View {
     @Binding var selected: PlayApp?
 
     @State var app: PlayApp
-    @State var iconURL: URL?
+    @State var appIcon: NSImage?
     @State var isList: Bool
     @State var hasPlayTools: Bool?
 
     var body: some View {
         Group {
             if isList {
                 HStack(alignment: .center, spacing: 0) {
-                    AsyncImage(url: iconURL) { image in
-                        image
-                            .resizable()
-                            .aspectRatio(contentMode: .fit)
-                            .frame(width: 30, height: 30)
-                            .cornerRadius(7.5)
-                            .shadow(radius: 1)
-                            .padding(.horizontal, 15)
-                            .padding(.vertical, 5)
-                    } placeholder: {
-                        ProgressView()
-                            .progressViewStyle(.circular)
-                            .frame(width: 60, height: 60)
+                    Group {
+                        if let image = appIcon {
+                            Image(nsImage: image)
```

Comment:
This takes significantly longer to load than `AsyncImage`/`ImageCache`. When switching to App Library `Image Cache` takes around ~0.5-1 seconds whereas this system takes around ~4-5 seconds.

Reply:
That's on first load I assume, in my tests it was same as ImageCache